resources/js/stores/useAttributes.js

```diff
@@ -37,4 +38,8 @@ export const attributes = computedAsync(
     { lazy: true, shallow: false },
 )
 
+window.attributeLabel = computed((attributeCode) => {
```

Comment:
If this works it's fine, i'd expect the return value would need to be a computed value instead of the function itself for it to work properly